apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/service/DollyBestillingService.java

```diff
@@ -214,4 +216,9 @@ protected Optional<DollyPerson> prepareDollyPerson(BestillingProgress progress)
 
         return nonNull(dollyPerson) ? Optional.of(dollyPerson) : Optional.empty();
     }
+
+    public static List<String> getEnvironments(String miljoer){
+
+        return new ArrayList<>(isNotBlank(miljoer) ? (List.of(miljoer.split(","))) : emptyList());
```

Comment:
trenger vi "new ArrayList" her?

Reply:
Tror du har rett. List.of gir immutable, mens ArrayList er mutable. Kan forsøke uten og hvis IllegalOperationException blir kastet et stede lengre nede i kjeden endrer jeg tilbake. 👍